optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -502,11 +502,11 @@ The following example penalizes computers which have at least one process runnin
 ----
 
 Note the use of the `ifExists()` building block.
-On `UniConstraintStream`, the `ifExistsOther()` building block is also available, useful in situations where your
-`from()` constraint match type is the same as your `ifExists()` type.
+On `UniConstraintStream`, the `ifExistsOther()` building block is also available which is useful in situations where the
+`from()` constraint match type is the same as the `ifExists()` type.
 
-Conversely, the `ifNotExists()` (and `ifNotExistsOther()` on `UniConstraintStream`) are available to achieve the
-opposite effect:
+Conversely, the `ifNotExists()` building block is available (as well as the `ifNotExistsOther()` building block on
```

Comment:
Good, but are you missing a couple of words:
Conversely, if the `ifNotExists()` building block is available (as well as the `ifNotExistsOther()` building block on `UniConstraintStream`) this achieves the opposite effect:

Reply:
Sorry, I merged too soon. @triceo Can you fix in a separate PR (if needed)?